charts/configcat-proxy/templates/configmap.yaml

```diff
@@ -1,10 +1,11 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Release.Name }}-configmap
```

Comment:
We want to use the `.fullname` helper as it will always give us the similar naming scheme even if the release name was overwritten. You can see this in the `_helpers.tpl` file.